docs/reference/api/connectivity/bluetooth/BatteryService.md

```diff
@@ -1,11 +1,19 @@
 ## BatteryService
 
-[Add description here.]
+It is often a requirement for devices operating on battery to report the battery charge level to the user.
+
+The <a href="https://www.bluetooth.org/docman/handlers/downloaddoc.ashx?doc_id=245138" target="_blank">Bluetooth Battery Service</a> defines how to expose a battery charge level through a BLE link. It allows a client - usually a smartphone application - of a device to read the current battery charge level and follow its evolution.
+
+The BatteryService class implements the Bluetooth Battery Service as defined by the Bluetooth body. Makers of BLE devices operating on battery can use the API to expose interoperably the charge level of their products.
```

Comment:
Bluetooth body or SIG (special interest group) as they refer to themselves as?

Reply:
I've replaced Bluetooth body by Bluetooth SIG.